content/ru/docs/intro/_index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Введение"
+weight: 1
+aliases: ["/docs/using_helm/"]
```

Comment:
You can remove all the aliases lines. Those are only necessary for English docs that got moved to a different location. They don't apply to new translations.